encord/user_client.py

```diff
@@ -581,7 +575,7 @@ def __get_recursive_image_paths(images_directory_path: Path) -> List[Path]:
 
     def __upload_cvat_images(
         self, images_paths: List[Path], used_base_path: Path, dataset_name: str
-    ) -> Tuple[str, Dict[str, str]]:
+    ) -> Tuple[str, Dict[str, dict[str, str]]]:
```

Comment:
thinking:
Why doesn't this need to be:
```suggestion
    ) -> Tuple[str, Dict[str, Dict[str, str]]]:
```?
I would have thought it would have complained 🤔